src/JsonSchema/Validator.php

```diff
@@ -27,10 +27,10 @@ class Validator extends BaseConstraint
 {
     public const SCHEMA_MEDIA_TYPE = 'application/schema+json';
 
-    public const ERROR_NONE                    = 0x00000000;
-    public const ERROR_ALL                     = 0xFFFFFFFF;
-    public const ERROR_DOCUMENT_VALIDATION     = 0x00000001;
-    public const ERROR_SCHEMA_VALIDATION       = 0x00000002;
+    public const ERROR_NONE                    = 0;
+    public const ERROR_ALL                     = -1;
```

Comment:
`self::ERROR_DOCUMENT_VALIDATION | self::ERROR_SCHEMA_VALIDATION` also should work here no? Or just hardcode `3` because frankly this is very unlikely to change, so if a new type is added with 4 then you just gotta update the ERROR_ALL.. Doesn't sound that hard 🤷🏻 


```suggestion
    public const ERROR_ALL                     = self::ERROR_DOCUMENT_VALIDATION | self::ERROR_SCHEMA_VALIDATION;
```

Reply:
It makes more sense to me to use -1 for an E_ALL level. Like `error_reporting(-1)` is a common way to turn on all error levels for the engine...

Reply:
Yup, works for me with -1 too.. Not sure why PHPStan complains though, but I tried on my machine and wasn't able to quiet it down. Seems like the 0xFFFFFFFF makes it give up on some checks.

Reply:
Phpstan complains because it doesn’t support int-ranges from binary & and | operator yet.

Reply:
Ok thanks, i thought i saw this work but must remember it wrong then :)